io/disk/iozone.py.data/README

```diff
@@ -0,0 +1,11 @@
+IOzone is a filesystem benchmark tool. The benchmark generates and measures
+a variety of file operations. Iozone has been ported to many machines and
+runs under many operating systems.
+
+Inputs Needed in yaml file:
+---------------------------
+args - Arguements with which iozone command is to be run.
+directory - Directory from which iozone test is executed.
+previous_run - Absolute path of raw_output file of any previously ran iozone
```

Comment:
This should probably be `previous_results`, it's a minor change, I'll fix it during merge.